src/app/app-routing.module.ts

```diff
@@ -19,10 +19,27 @@ const routes: Routes = [
   redirectTo: 'pages/LandingPage',
   pathMatch: 'full'
 },
-   {
-        path: '**',
-        redirectTo: ''
-					},
+   
+{ 
+  path: '/', 
+  redirectTo: 'pages/virtual-room',
+  pathMatch: 'full'
+},
+{ 
+  path: '/virtual-room', 
+  redirectTo: 'pages/virtual-room',
+  pathMatch: 'full'
+},
+{ 
+  path: 'pages/virtual-room', 
+  redirectTo: 'pages/virtual-room',
+  pathMatch: 'full'
+},
```

Comment:
You'll need to specify the component that should render based on the route. Since this is the route that matches the directory path, it'll be best to add it here as shown below

```suggestion
{ 
  path: 'pages/virtual-room', 
  component: VirtualRoomComponent,
  pathMatch: 'full'
},
```

The same will need to be done for the LandingPage Component. Let me know if this needs further explanation

Reply:
Added the component path